### PR TITLE
[DT-219][risk=no] Add domain token param to command

### PR DESCRIPTION
**Pull request by dolbeew**

### CODE REVIEW

File: api/libproject/devstart.rb [hunk 1]
```diff
@@ -833,6 +833,11 @@ def build_review_all_events(cmd_name, *args)
       ->(opts, v) { opts.bq_dataset = v},
       "BQ dataset. Required."
   )
+  op.add_option(
+    "--domain-token [domain-token]",
+    ->(opts, v) { opts.domain_token = v},
+    "Generate for domain-token. Required."
+  )
```
Comment (freemabd) [inline api/libproject/devstart.rb:840]: you also need to pass the arg in the script call below on line 847

File: api/libproject/devstart.rb [hunk 2]
```diff
@@ -833,6 +833,11 @@ def build_review_all_events(cmd_name, *args)
...
 
   op.add_validator ->(opts) { raise ArgumentError unless opts.bq_project and opts.bq_dataset}
```
Comment (freemabd) [inline api/libproject/devstart.rb:842]: If it's required, then it needs to be validated here

**Review by freemabd — COMMENTED** [review]
few comments

File: api/libproject/devstart.rb (hunk 1 above)
Comment (dolbeew) [reply api/libproject/devstart.rb:840]: Done, thanks.

File: api/libproject/devstart.rb (hunk 2 above)
Comment (dolbeew) [reply api/libproject/devstart.rb]: Done.

**Review by freemabd — APPROVED** [review]

**Pull request merged**